<ui version="4.0" >
 <class>ProjectConfig</class>
 <widget class="QWidget" name="ProjectConfig" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>412</width>
    <height>232</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <widget class="QLabel" name="project" >
     <property name="text" >
      <string>Default project directory:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="PathSelector" name="projectDirectory" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize" >
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="source" >
     <property name="text" >
      <string>Default source directory:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="PathSelector" name="sourceDirectory" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize" >
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QLabel" name="build" >
     <property name="text" >
      <string>Default build directory:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" >
    <widget class="PathSelector" name="buildDirectory" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize" >
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2" >
    <widget class="Line" name="line" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="0" >
    <widget class="QLabel" name="last" >
     <property name="text" >
      <string>Open last project on startup:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" >
    <widget class="QRadioButton" name="lastProject" >
     <property name="layoutDirection" >
      <enum>Qt::LeftToRight</enum>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2" >
    <widget class="Line" name="line_2" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2" >
    <widget class="BuildSource" name="buildSource" />
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PathSelector</class>
   <extends>QWidget</extends>
   <header>pathselector.h</header>
  </customwidget>
  <customwidget>
   <class>BuildSource</class>
   <extends>QWidget</extends>
   <header>buildsource.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
